### ISSUE TITLE
rake task for slugging existing records not found

### ISSUE THREAD

**Issue by vinchi777** [digitalplaywright/mongoid-slug#212]
Using mongo-slug version 5.1.1
```
rake mongoid_slug:set
Don't know how to build task 'mongoid_slug:set'
```

**Comment by anuja-joshi** [comment]
@vinchi777 This task will be added to next gem release **mongoid-slug version 5.1.2** , you can wait for next version release or can use mongoid-slug directly from git repository's master branch.

**Comment by vinchi777** [comment]
oh i see thanks!

**Comment by dblock** [comment]
I released 5.2.0 this AM, sorry for the delay.

**Comment by vinchi777** [comment]
no problem. wow that was fast!
